src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -677,13 +910,14 @@ public void fileDownload(@PathVariable(value = "uniqueID") String uniqueID,
             }
         }
     }
-
+*/
     /**
      * Attempts to get the client IP address from the request.
      *
      * @param request The HttpServletRequest.
      * @return The client's IP address.
      */
+    /*
     private String getIpAddress(HttpServletRequest request) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)